apps/internal/mock/mock.go

```diff
@@ -90,11 +90,19 @@ func (c *Client) Do(req *http.Request) (*http.Response, error) {
 // CloseIdleConnections implements the comm.HTTPClient interface
 func (*Client) CloseIdleConnections() {}
 
-func GetAccessTokenBody(accessToken, idToken, refreshToken, clientInfo string, expiresIn int) []byte {
+func GetAccessTokenBody(accessToken, idToken, refreshToken, clientInfo string, expiresIn, refreshIn int) []byte {
+	// Start building the body with the common fields
 	body := fmt.Sprintf(
 		`{"access_token": "%s","expires_in": %d,"expires_on": %d,"token_type": "Bearer"`,
 		accessToken, expiresIn, time.Now().Add(time.Duration(expiresIn)*time.Second).Unix(),
 	)
+
+	// Conditionally add the "refresh_in" field if refreshIn is provided
+	if refreshIn > 0 {
+		body += fmt.Sprintf(`, "refresh_in": %d`, refreshIn)
```

Comment:
is `refresh_in` always a number, or could it be a string?

Reply:
Always a number, same as `expires_in`

Reply:
expires_in can be a string though. I'm referring to JSON value types here i.e., is refresh_in always a number like `42` or can it be a string like `"42"`

Reply:
Updated, tested it and it works fine both ways :)